PrebidMobile/PrebidMobileCore/src/main/java/org/prebid/mobile/core/TargetingParams.java

```diff
@@ -112,6 +115,48 @@ static void fetchLocationUpdates(Context appContext) {
 
     //region Public APIs
 
+    public static void setGDPRConsentStrings(Context context, ArrayList<String> strings) {
+        if (strings != null && context != null) {
+            SharedPreferences pref = PreferenceManager.getDefaultSharedPreferences(context);
+            SharedPreferences.Editor editor = pref.edit();
+            editor.putString("Prebid_GDPR_consent_strings", strings.toString());
+            editor.apply();
+        }
+    }
+
+    public static ArrayList<String> getGDPSConsentStrings(Context context) {
```

Comment:
consent is sent has string object in openRTB request... why are you splitting it agin to array here?